handlers/handlers.go

```diff
@@ -0,0 +1,377 @@
+// handlers/handlers.go
+package handlers
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"log"
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/dizhechko/todo-list-server/database"
+	"github.com/dizhechko/todo-list-server/scheduler"
+	"github.com/dizhechko/todo-list-server/settings"
+)
+
+/*
+	type Claims struct {
+		//Exp      int64  `json:"exp"`
+		Checksum string `json:"checksum"`
+		jwt.StandardClaims
+	}
+*/
+type TaskID struct {
+	ID int64 `json:"id"`
+}
+
+// errorJSON возвращает json-строку с ошибкой
+func errorJSON(err error) string {
+	jsonError, err := json.Marshal(map[string]string{"error": err.Error()})
+	if err != nil {
+		println(err)
+		return ""
+	}
+	return string(jsonError)
+}
+
+// NextDateHandler получает следующую дату повторения задачи по параметрам now, date, repeat
+func NextDateHandler(w http.ResponseWriter, r *http.Request) {
+	//Получаем параметры из запроса
+	now := r.FormValue("now")
+	date := r.FormValue("date")
+	repeat := r.FormValue("repeat")
+
+	//Вычисляем следующую дату
+	nextDate, err := scheduler.NextDate(now, date, repeat)
+	if err != nil {
+		log.Printf("NextDateHandler: %v %v %v %v; error: %v\n", now, date, repeat, nextDate, err)
+		w.WriteHeader(http.StatusBadRequest)
+		_, _ = w.Write([]byte(""))
```

Comment:
Отлично сделано, что ошибка идет в лог, а пользователю  пустой ответ, но можно и заглушку
```suggestion
		_, _ = w.Write([]byte("Internal server error"))
```